fleetspeak/src/server/server.go

```diff
@@ -153,6 +159,11 @@ func MakeServer(c *spb.ServerConfig, sc Components) (*Server, error) {
 
 	s.dataStore.RegisterMessageProcessor(s.serviceConfig)
 
+	if s.healthCheckListener != nil {
+		http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {})
```

Comment:
It would be better to create a http.Server instance, instead of registering "global" handlers.

(http.HandleFunc modifies a global http.Server instance).